### ISSUE TITLE
app forcefully closed with this error on os version < 21

### ISSUE THREAD

**Issue by karan101292** [pocesar/react-native-sim-data#2]
app forcefully closed with this error on `os version < 21`

`com.facebook.react.common.JavascriptException: C++ Exception in 'NativeModules': java.lang.NoSuchMethodError: android.telephony.TelephonyManager.getPhoneCount`

react native - v44

**Comment by pocesar** [comment]
although TelephonyManager is available since API 1, it doesn't exists in the current code

**Comment by karan101292** [comment]
Yes `SubscriptionManager`  needs API 22. so in that case, this module will not work in phones with API less than 22. or is there any workaround?

**Comment by pocesar** [comment]
you can use `TelephonyManager.getLine1Number` but the person needs to have set his own number on the phone in "phone identity" and most people don't
